packages/esbuild/_README.md

```diff
@@ -17,16 +17,17 @@ yarn add -D @bazel/esbuild
 
 Add an `http_archive` fetching the esbuild binary for each platform that you need to support. 
 
+<!-- IMPORTANT: Keep this in sync with ./esbuild_repo.bzl -->
```

Comment:
Is this comment ignored by the docs generation?

Reply:
Good point, removed.